druid-shell/src/lib.rs

```diff
@@ -81,3 +81,6 @@ pub use window::{
 };
 
 pub use keyboard_types;
+
+#[cfg(target_os = "linux")]
+pub use application::LinuxApplicationExt;
```

Comment:
how we should expose these is a good question.

I like the pattern in `std` of having the `std::os` module, which contains `std::os::unix`, `std::os:windows`, etcetera; I could imagine having something like that?

hmhm:

If we made the `platform` module public we could put these in the particular root modules of the specific platforms, but that might get a bit messy.

Maybe what I would prefer instead would be that we have an `ext` module, and then `ext::linux`? Another option would be to rename `platform` to `backend` or something, and then we could use `platform` for this, which feels like a better name...

Reply:
I like `platform::LinuxApplicationExt` or `platform::linux::LinuxApplicationExt`. renaming existing `platform` to `backend` sound reasonable.

I would prefer shorter use statement:
`use druid::shell::platform::LinuxApplicationExt;`
over
`use druid::shell::platform::linux::LinuxApplicationExt;`

Reply:
we could also have `druid::platform::linux` that reexports things from shell?

Now should we generally distinguish between `platform::gtk` and `platform::x11`?

Reply:
I guess `platform::linux::{GtkApplicationExt, X11ApplicationExt}`. X11 and Wayland should be different though